build.gradle

```diff
@@ -55,13 +55,27 @@ configurations {
 }
 
 repositories {
+    flatDir { dirs 'libs' }
     mavenLocal()
     maven { url = "https://maven.covers1624.net/" }
     maven { url = "https://dvs1.progwml6.com/files/maven" }
     maven { url = "https://maven.cil.li/" }
     maven { url = "https://maven.theillusivec4.top/" }
     maven { url = "https://squiddev.cc/maven/" }
     maven { url = "https://maven.blamejared.com/" }
+    maven {
+        url "https://maven.architectury.dev"
```

Comment:
Ideally, I would like to avoid having an arch dep. You should be able to just dep on the OC2 api, which _should not_ require an arch dep.